partner_education_views/models/education_level.py

```diff
@@ -7,3 +7,16 @@ class ResPartnerEducationLevel(models.Model):
 
     name = fields.Integer(string="Education level")
     description = fields.Char(string="Description")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducationLevel, self).name_get()
+        data = []
+        for edu_lvl in self:
```

Comment:
String concatenation is pretty expensive, its cheaper to build a list and then do "".join(my_list)
In this case it does probably not matter but think of it in the future.